src/extensions/replay/sessionrecording-utils.ts

```diff
@@ -32,12 +32,25 @@ export function estimateSize(sizeable: unknown): number {
 export const replacementImageURI =
     'data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIGhlaWdodD0iMTYiIHZpZXdCb3g9IjAgMCAxNiAxNiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHJlY3Qgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2IiBmaWxsPSJibGFjayIvPgo8cGF0aCBkPSJNOCAwSDE2TDAgMTZWOEw4IDBaIiBmaWxsPSIjMkQyRDJEIi8+CjxwYXRoIGQ9Ik0xNiA4VjE2SDhMMTYgOFoiIGZpbGw9IiMyRDJEMkQiLz4KPC9zdmc+Cg=='
 
+// values copied from rrweb's EventType and IncrementalSource enums
 export const FULL_SNAPSHOT_EVENT_TYPE = 2
-export const META_EVENT_TYPE = 4
 export const INCREMENTAL_SNAPSHOT_EVENT_TYPE = 3
+export const META_EVENT_TYPE = 4
+export const CUSTOM_EVENT_TYPE = 5
 export const PLUGIN_EVENT_TYPE = 6
 export const MUTATION_SOURCE_TYPE = 0
 
+export const MUTATION_SOURCE = 0
+export const MOUSE_MOVE_SOURCE = 1
+export const MOUSE_INTERACTION_SOURCE = 2
+export const SCROLL_SOURCE = 3
+export const VIEWPORT_RESIZE_SOURCE = 4
+export const INPUT_SOURCE = 5
+export const TOUCH_MOVE_SOURCE = 6
+export const MEDIA_INTERACTION_SOURCE = 7
+export const STYLE_SHEET_RULE_SOURCE = 8
+export const DRAG_SOURCE = 12
+
```

Comment:
we had some of these defined here, but not all the ones we use
so we may as well have one way of doing things